tools/deployment/package/docker-compose.yaml

```
@@ -114,3 +114,28 @@ services:
+    healthcheck:
+      <<: *healthcheck_defaults
+      test: [
+        "CMD",
+        "curl",
+        "-f",
+        "http://${CLP_MCP_CONTAINER_HOST:-0.0.0.0}:${CLP_MCP_CONTAINER_PORT:-8000}/health"
+      ]
+    command: [
+      "python3", "-u",
+      "-m", "clp_mcp_server.clp_mcp_server",
+      "--host","${CLP_MCP_CONTAINER_HOST:-0.0.0.0}",
```

YAMLlint (1.37.1)

[warning] too few spaces after comma

(commas)